metabase/metabase
# Temporal unit parameters in dashboards

Epic https://github.com/metabase/metabase/issues/42118
Demo https://www.loom.com/share/13078400d1724220a5500669e5fe064e (autowiring on adding a new parameter was reverted, but regular one works).

Allows to add a temporal unit parameter that can be mapped to breakout columns in MBQL queries that allow changing the unit. 

How to verify:
- Add a question with a temporal breakout to a dashboard
- Add a "unit of time" parameter to the dashboard
- Connect it to the card
- Make sure you can change the temporal granularity 
- Make sure you can open the question when there is a filter and change is propogated to the QB

Tests tbd in a separate PR. Now there is a single e2e test.
This is currently very raw and I expect issues until the testing plan is implemented.
